### PR TITLE
DSNC-3778: Anaconda Individual Edition 2021.11 docker images

### PR DESCRIPTION
**Pull request by adipietro18**
If merged, we can tag `anaconda3-2021.11` to build & deploy the images.

### CODE REVIEW

File: .github/workflows/anaconda_alpine_ci.yml [hunk 1]
```diff
@@ -16,18 +18,50 @@ jobs:
...
       - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
 
+      - name: Login to DockerHub
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        if: github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags')
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Login to Public ECR
```
Comment (dbast) [inline .github/workflows/anaconda_alpine_ci.yml:28]: Does public.ecr.aws/y0o4y9o3/anaconda3 exist? Afaik we only push the anaconda-pkg-build images to ECR to have rate limits during package builds.

File: .github/workflows/anaconda_debian_ci.yml [hunk 2]
```diff
@@ -18,15 +20,25 @@ jobs:
 
       - name: Login to DockerHub
         uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
-        if: github.ref == 'refs/heads/master'
+        if: github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags')
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Login to Public ECR
```
Comment (dbast) [inline .github/workflows/anaconda_debian_ci.yml:28]: Does public.ecr.aws/y0o4y9o3/anaconda3 exist? Afaik we only push the anaconda-pkg-build images to ECR to have rate limits during package builds.

File: anaconda3/alpine/Dockerfile [hunk 3]
```diff
@@ -49,8 +49,8 @@ RUN ALPINE_GLIBC_BASE_URL="https://github.com/sgerrand/alpine-pkg-glibc/releases
 FROM alpine-glibc
 
 LABEL maintainer="Anaconda, Inc"
```
Comment (dbast) [inline anaconda3/alpine/Dockerfile:51]: Do we want to update to `alpine:3.15 as alpine-glibc`, see beginning of this file. The miniconda images are at 3.14.

File: anaconda3/amazonlinux/Dockerfile [hunk 4]
```diff
@@ -8,7 +8,7 @@ ENV PATH /opt/conda/bin:$PATH
 RUN yum install -y wget bzip2 ca-certificates libglib2.0-0 libxext6 libsm6 libxrender1 git mercurial subversion && \
```
Comment (dbast) [inline anaconda3/amazonlinux/Dockerfile:8]: The amazon image could be also updated to `arm64v8/amazonlinux:2.0.20211005.0`, see beginning of the file. The anaconda-pkg-build image, just uses `amazonlinux:2`.

File: .github/workflows/anaconda_alpine_ci.yml (hunk 1 above)
Comment (adipietro18) [reply .github/workflows/anaconda_alpine_ci.yml:28]: It seemed to [exist](https://gallery.ecr.aws/y0o4y9o3/anaconda3) but did not have anything pushed to it. I will remove the ECR images.

**Review by dbast — APPROVED** [review]

**Pull request merged**